.github/workflows/frameworks.yml

```diff
@@ -0,0 +1,35 @@
+name: Frameworks tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  boto3:
+    name: boto3 with Python ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
You need to fetch more of the git history for setuptool-scm to build the right version. This explains the version issue you see below.